Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,210 @@
+ is enabled
+  * `replicationState` should be `primary` for all the PVC’s on
+   the secondary site.
+* [Check VolumeReplication CR status](rbd-mirroring.md#checking-replication-status) to verify if the image is marked `primary` on the secondary site.
+* Once the Image is marked as `primary`, the PVC is now ready
+ to be used. Now, we can scale up the applications to use the PVC.
+
+>:memo: **WARNING**: In Async Disaster recovery use case, we don't get
+> the complete data.
+> We will only get the crash-consistent data based on the snapshot interval time.
+
+## Disaster Recovery
+
+> Use cases: Natural disasters, Power failures, System failures, and crashes, etc.
+
+### Failover (abrupt shutdown)
+
+In case of Disaster recovery, create VolumeReplication CR at the Secondary Site.
+ Since the connection to the Primary Site is lost, the operator automatically
+ sends a GRPC request down to the driver to forcefully mark the dataSource as `primary`
+ on the Secondary Site.
+
+* If you are manually creating the PVC and PV on the secondary cluster, remove
+ the claimRef section in the PV objects. (See [this](async-disaster-recovery.md#restore-the-backup-on-cluster-2) for details)
+* Create the storageclass, PVC, and PV objects on the secondary site.
+* As you are creating the static binding between PVC and PV, a new PV won’t be
+ created here, the PVC will get bind to the existing PV.
+* [Create the VolumeReplicationClass](rbd-mirroring.md#create-a-volume-replication-class-cr) and [VolumeReplication CR](rbd-mirroring.md#create-a-volumereplication-cr) on the secondary site.
+* [Check VolumeReplication CR status](rbd-mirroring.md#checking-replication-status) to verify if the image is marked `primary` on the secondary site.
+* Once the Image is marked as `primary`, the PVC is now ready to be used. Now,
+ we can scale up the applications to use the PVC.
+
+### Failback (post-disaster recovery)
+
+Once the failed cluster is recovered on the primary site and you want to failback
+ from secondary site, follow the below steps:
+
+* Scale down the running applications(if any) on the primary site.
+ Ensure that all persistent volumes in use by the workload are no
+ longer in use on the primary cluster.
+* [Update VolumeReplication CR](rbd-mirroring.md#create-a-volumereplication-cr) replicationState
+ from `primary` to `secondary` on the primary site.
+* Scale down the applications on the secondary site.
+* [Update VolumeReplication CR](rbd-mirroring.md#create-a-volumereplication-cr) replicationState state from `primary` to
+ `secondary` in secondary site.
+* On the primary site, [verify the VolumeReplication status](rbd-mirroring.md#checking-replication-status) is marked as
+ volume ready to use.
+* Once the volume is marked to ready to use, change the replicationState state
+ from `secondary` to `primary` in primary site.
+* Scale up the applications again on the primary site.
+
+## Appendix
```

Comment:
Why is backup & restore in an appendix? Shouldn't backup & restore be a 2nd-level heading?